tests/test_profiler.py

```diff
@@ -58,8 +48,8 @@ def _get_duration(profile):
     # different environments have different precision when it comes to time.sleep()
     # see: https://github.com/PyTorchLightning/pytorch-lightning/issues/796
     a_duration = _get_duration(p.profiled_actions["a"])
-    _assert_almost_equal(a_duration, [4], tol=0.5)
+    np.testing.assert_allclose(a_duration, [4], atol=0.5)
```

Comment:
is there a reason you chose different values for `atol` for the simple profiler and the advanced profiler?

Reply:
Comes from CI tests...

Reply:
based on your other comment it seems like you may have intended to set `atol=0.5` for both `test_simple_profiler()` and `test_advanced_profiler()`. is that correct?

Reply:
Yes, we can set any tolerance for each independently...

Reply:
ah i figured out my confusion. i was wondering why we would want to have different values for tolerance between the tests for the two profilers. then i remembered that the `SimpleProfiler` test is measuring against each individual run of an action whereas the `AdvancedProfiler` test is measuring the total duration for each action. so since they're measuring two different things it makes sense why we might want to have different tolerances, sorry about that 😄